desktop/core/src/desktop/js/parse/sql/generic/jison/sql.jisonlex

```diff
@@ -36,16 +36,20 @@
 'ASC'                                      { return 'ASC'; }
 'BETWEEN'                                  { this.begin('between'); return 'BETWEEN'; }
 'BIGINT'                                   { return 'BIGINT'; }
+'BINARY'                                   { return 'BINARY'; }
```

Comment:
Had to manually specify all the possible types here otherwise they weren't treated in a case-sensitive manner. (This file starts with `%options case-insensitive flex` so I think that's what makes it work if listed here)